pages/availability/index.tsx

```diff
@@ -249,6 +253,59 @@ export default function Availability(props) {
                                                     minutes
                                                 </div>
                                             </div>
+                                        </div>    
+                                        
+                                        <div className="mb-4">
+                                            <span className="block text-sm font-medium text-gray-700">
+                                                Date range
+                                            </span>
+                                            <div className="flex space-between">
+                                                <label className="block text-sm mb-1 mr-4 font-medium text-gray-700">
+                                                    <input
+                                                        type="radio"
+                                                        className="form-radio"
+                                                        onChange={() => setTimeRange('specific')}
+                                                        name="dateRange"
+                                                        value="specific"
+                                                        checked={timeRange === 'specific'}
+                                                    />
+                                                    <span className="ml-2 text-xs">Range</span>
+                                                </label>
```

Comment:
Codacy has a fix for the issue: [Delete `························`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
                        </label>
```